blocks/library/paragraph/index.js

```diff
@@ -302,9 +307,15 @@ const schema = {
 	textColor: {
 		type: 'string',
 	},
+	customTextColor: {
```

Comment:
This proliferation of attributes is excessive. Particularly to my previous comment of existing blocks, can't we have a single attribute, the hex of the color? Shouldn't matter if it's custom or not. If it aligns with an existing hex for a named color, we can identify it by that. I thought this is what we were doing with font size, though I've not kept up with what we decided for #5269.